src/main/java/uk/gov/moj/sdt/config/WebSecurityConfig.java

```diff
@@ -16,10 +16,6 @@ public class WebSecurityConfig {
 
     @Bean
     public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
-
-        http.authorizeHttpRequests()
-            .anyRequest().permitAll();
-
         return http.build();
```

Comment:
This does appear to need the following configuration in it to allow access to SDT:
`http.csrf().disable().formLogin().disable().logout().disable().authorizeRequests().anyRequest().permitAll();`